styles/styles.css

```diff
@@ -75,5 +76,9 @@ body{
 }
 .hotel-foto span{
     color: #BFBFBF;
-    
+}
+.arrow-right{
+    position: absolute;
+    padding-top: 128px;
+    right: 79px;
```

Comment:
after replacing padding with margin at the previous comment it's better to replace padding-top with top value for position